api/src/modules/users/users.routes.ts

```diff
@@ -27,6 +28,15 @@ export const users = async (fastify: FastifyInstance) => {
     preValidation: [fastify.verifyPassword],
   });
 
+  fastify.route({
+    method: 'PUT',
+    url: '/users/avatar',
+    // TODO: Add schema for avatar upload,
+    handler: handlePutUserAvatar,
```

Comment:
Schema validation was more complicated than the checks on the hook that I've created, It may be a good idea to make one for consistency purpose in futur